metasrv/Cargo.toml

```diff
@@ -29,7 +30,7 @@ common-base = { path = "../common/base" }
 common-exception = { path = "../common/exception" }
 common-grpc = { path = "../common/grpc" }
 common-macros = { path = "../common/macros" }
-common-meta-api = { path = "../common/meta/api" }
+common-meta-api = { path = "../common/meta/api", features = ["create_with_drop_time"] }
```

Comment:
This will always enable feature `create_with_drop_time`.

We should add `optional = true` here.